pages/design/atomic-template-empty.md

```diff
@@ -1,35 +1,50 @@
 ---
+id: 
 name:
 rule_type: atomic
 
 description: |
   This rule checks ...
 
-success_criterion:
-  - x.x.x # (Name of Success Criterion)
-
-test_aspects:
+accessibility_requirements: # Remove whatever is not applicable
```

Comment:
@WilcoFiers 
If there are no requirements, like in this PR https://github.com/act-rules/act-rules.github.io/pull/523/files#diff-f1ba9ad105ee9aa5ee46dfd067f72397R7, is it worth adding `accessibility_requirements`?